tests/test_apply.py

```diff
@@ -563,6 +565,7 @@ def test_expval_single_wire_no_parameters(
 
         assert np.isclose(res, expected_output, atol=tol, rtol=0)
 
+    @pytest.mark.xfail(ld._new_API, reason="Old API required")
```

Comment:
This is because we don't have samples/shots yet, right?

Reply:
Yes.

Reply:
Yes.